tools/dev.py

```diff
@@ -97,8 +97,8 @@ def shutdown():
     docker_compose_run(["down"])
 
 
-def run_server(display_remind: bool = False, run_frontend_in_production_mode: bool = False) -> None:
-    if run_frontend_in_production_mode:
+def run_server(display_remind: bool = False, run_frontend_in_prod_mode: bool = False) -> None:
```

Comment:
I'm not a big fan of such "shortenings", especially since we don't gain much from it. I suggest reverting this change.